examples/notebooks/spleen_segmentation_3d.ipynb

```diff
@@ -311,10 +311,10 @@
     "        loss.backward()\n",
     "        optimizer.step()\n",
     "        epoch_loss += loss.item()\n",
-    "        print('{}/{}, train_loss: {:.4f}'.format(step, len(train_ds) // train_loader.batch_size, loss.item()))\n",
+    "        print(f\"{step}/{len(train_ds) // train_loader.batch_size}, train_loss: {loss.item()}\")\n",
```

Comment:
The previous string only keeps 4 places after the decimal point.
Something like: `f"XXX{data: 4}"`

Thanks.